common/build/eslint-config-fluid/src/custom-rules/no-restricted-tags-imports.js

```diff
@@ -0,0 +1,77 @@
+/*!
+ * Copyright (c) Microsoft Corporation and contributors. All rights reserved.
+ * Licensed under the MIT License.
+ */
+
+module.exports = {
+	meta: {
+		type: "problem",
+		docs: {
+			description:
+				"This rule restricts imports from internal or non-public APIs. This to prevent accidental dependencies on internal, unstable or undocumented parts of the codebase.",
+			category: "Best Practices",
+		},
+		fixable: "code",
+		schema: [
```

Comment:
Is there any way for us to run a 1-time check on all the specified tags to ensure they're of the right form (e.g. starts with `@`)?

Reply:
Or if we can specify this declaratively in the schema somehow, but I'm guessing that isn't an option?

Reply:
Yeah, you can't specify it in the schema, but I added a check when pulling in the tags that throws an error if it isn't formatted appropriately.